internal/transform/operation.go

```diff
@@ -83,17 +83,18 @@ func TransformOperation(operation xdr.Operation, operationIndex int32, transacti
 	outputLedgerSequence := utils.GetLedgerSequence(ledgerCloseMeta)
 
 	transformedOperation := OperationOutput{
-		SourceAccount:       outputSourceAccount,
-		SourceAccountMuxed:  outputSourceAccountMuxed.String,
-		Type:                outputOperationType,
-		TypeString:          outputOperationTypeString,
-		TransactionID:       outputTransactionID,
-		OperationID:         outputOperationID,
-		OperationDetails:    outputDetails,
-		ClosedAt:            outputCloseTime,
-		OperationResultCode: outputOperationResultCode,
-		OperationTraceCode:  outputOperationTraceCode,
-		LedgerSequence:      outputLedgerSequence,
+		SourceAccount:        outputSourceAccount,
+		SourceAccountMuxed:   outputSourceAccountMuxed.String,
+		Type:                 outputOperationType,
+		TypeString:           outputOperationTypeString,
+		TransactionID:        outputTransactionID,
+		OperationID:          outputOperationID,
+		OperationDetails:     outputDetails,
+		ClosedAt:             outputCloseTime,
+		OperationResultCode:  outputOperationResultCode,
+		OperationTraceCode:   outputOperationTraceCode,
+		LedgerSequence:       outputLedgerSequence,
+		OperationDetailsJSON: outputDetails,
```

Comment:
What is the real difference between OperationDetails and OperationDetailsJSON

Reply:
I see. Are you going to drop individual fields subsequently and keep only the json blob?

Reply:
We could drop the `details` record eventually after we talk with partners and they move off of it. Oh I guess we would also need to move off of it as well

So yes in theory the end result would be
* drop `details`
* rename `details_json` to `details`